packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -193,3 +198,38 @@ export class GaugeMetric extends Metric<GaugeHandle>
     this.getHandle(labelSet).set(value);
   }
 }
+
+export class MeasureMetric extends Metric<MeasureHandle>
+  implements Pick<types.MetricUtils, 'record'> {
+  protected readonly _absolute: boolean;
+
+  constructor(
+    name: string,
+    options: MetricOptions,
+    private readonly _onUpdate: Function
+  ) {
+    super(
+      name,
+      options,
+      options.valueType === types.ValueType.DOUBLE
+        ? MetricDescriptorType.GAUGE_DOUBLE
+        : MetricDescriptorType.GAUGE_INT64
+    );
+
+    this._absolute = options.absolute !== undefined ? options.absolute : true; // Absolute default is true
```

Comment:
Another approach, what if we make `absolute` as a mandatory attribute in SDK (opentelemetry-metrics/src/types.ts) similar to other attributes. Most of these attributes are [optional](https://github.com/open-telemetry/opentelemetry-js/blob/668c3aa833707034569f3eb46f20fa2a9c0c0170/packages/opentelemetry-types/src/metrics/Metric.ts#L23) in `opentelemetry-types` but to make life easier, we made it [mandatory in the SDK](https://github.com/open-telemetry/opentelemetry-js/blob/173cab2a03d7ac8835b325c03296ccf0a3233ba3/packages/opentelemetry-metrics/src/types.ts#L21) with default values.